app/views/browse/build-config.html

```diff
@@ -408,7 +408,17 @@ <h3>Triggers <span class="pficon pficon-warning-triangle-o" ng-if="!(secretsVers
                               <dt>Generic Webhook URL:
                               </dt>
                               <dd>
-                                <copy-to-clipboard clipboard-text="buildConfig.metadata.name | webhookURL : trigger.type : trigger.generic : project.metadata.name : webhookSecrets"></copy-to-clipboard>
+                                <div ng-if="trigger.generic | isWebhookAvailable : webhookSecrets">
```

Comment:
Isn't this potentially a problem for all webhook types, not only generic webhooks?

Reply:
yeah its definitely problem of all the webhook triggers... this is just a WIP